parser.y

```diff
@@ -574,6 +574,7 @@ import (
 	hintEnablePlanCache	"ENABLE_PLAN_CACHE"
 	hintUsePlanCache	"USE_PLAN_CACHE"
 	hintReadConsistentReplica	"READ_CONSISTENT_REPLICA"
+	hintQBName	"QB_NAME"
```

Comment:
I'm not against using `QBName` in `ast.go`. But could we use the full name in `parser.y` to make its mean more clear?

Reply:
But full name would make using it harder beacause `queryblock` has 10 chars, it is reasonable trade off I think.

Reply:
I mean change `hintQBName "QB_NAME"` to `hintQBName "QUERY_BLOCK_NAME"`.

Reply:
It is also what I means, It contains 8 + 1 more chars that `QB_NAME`.

Reply:
All right...